node/src/node_configurator/unprivileged_parse_args_configuration.rs

```diff
@@ -479,30 +477,29 @@ fn configure_accountant_config(
     config: &mut BootstrapperConfig,
     persist_config: &mut dyn PersistentConfiguration,
 ) -> Result<(), ConfiguratorError> {
+    let payment_thresholds = process_combined_params(
+        "payment-thresholds",
+        multi_config,
+        persist_config,
+        |str: &str| PaymentThresholds::try_from(str),
+        |pc: &dyn PersistentConfiguration| pc.payment_thresholds(),
+        |pc: &mut dyn PersistentConfiguration, curves| pc.set_payment_thresholds(curves),
+    )?;
+    let scan_intervals = process_combined_params(
+        "scan-intervals",
+        multi_config,
+        persist_config,
+        |str: &str| ScanIntervals::try_from(str),
+        |pc: &dyn PersistentConfiguration| pc.scan_intervals(),
+        |pc: &mut dyn PersistentConfiguration, intervals| pc.set_scan_intervals(intervals),
+    )?;
     let suppress_initial_scans =
         value_m!(multi_config, "scans", String).unwrap_or_else(|| "on".to_string()) == *"off";
-
-    let accountant_config = AccountantConfig {
-        scan_intervals: process_combined_params(
-            "scan-intervals",
-            multi_config,
-            persist_config,
-            |str: &str| ScanIntervals::try_from(str),
-            |pc: &dyn PersistentConfiguration| pc.scan_intervals(),
-            |pc: &mut dyn PersistentConfiguration, intervals| pc.set_scan_intervals(intervals),
-        )?,
-        payment_thresholds: process_combined_params(
-            "payment-thresholds",
-            multi_config,
-            persist_config,
-            |str: &str| PaymentThresholds::try_from(str),
-            |pc: &dyn PersistentConfiguration| pc.payment_thresholds(),
-            |pc: &mut dyn PersistentConfiguration, curves| pc.set_payment_thresholds(curves),
-        )?,
-        suppress_initial_scans,
-        when_pending_too_long_sec: DEFAULT_PENDING_TOO_LONG_SEC,
-    };
-    config.accountant_config_opt = Some(accountant_config);
+    let when_pending_too_long = DEFAULT_PENDING_TOO_LONG_SEC;
+    config.payment_thresholds_opt = Some(payment_thresholds);
+    config.scan_intervals_opt = Some(scan_intervals);
+    config.suppress_initial_scans_opt = Some(suppress_initial_scans);
+    config.when_pending_too_long_opt = Some(when_pending_too_long);
```

Comment:
Actually, now I realized that this value also looks silly if you wrap it into an Option, assuming you're gonna take it out later, exchanging it for None in the end. But the value itself is just a single number so it's Copy. I'd prefer not to wrap it then because copying it is not a big deal...